docs/user-guide/dual-mesh.ipynb

```diff
@@ -0,0 +1,353 @@
+{
```

Comment:
<p>"When constructing the dual mesh paired with a node centered variable, the data becomes face centered.""</p><p> </p><p>An important note here should be that computing the dual of a node centered variable allows us to visualize the data as polygons (which you already showcase) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/859/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>